roles/sqs_queue/tasks/destroy.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: List SQS Queues
+  command: aws sqs list-queues --queue-name-prefix "{{ sqs_dead_letter_queue.name }}.fifo"
+  set_fact:
+    queue_to_remove: "{{ command_output.stdout }}"
```

Comment:
What do you do when there is more than one returned?

Reply:
the `.fifo` at the end of the queue prefix achieves what you were suggesting about adding something after the stack name. In effect we are always listing by the full queue name, however, I have put the stack name at the front of the queue name now. Commit to follow on gitlab